### PR TITLE
Save point

### PR DESCRIPTION
**Pull request by ZoeChiri**
um hello!

### CODE REVIEW

**Comment by ZoeChiri** [comment]
uhhh did it work

File: pkg/analysis/analysis.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	if err != nil{
+		return house.House{}, err
+	}
+
+	var gov House.house
+	err = json.Unmarshal(r, &gov)
+	if err != nil {
+		fmt.Println("error:", err)
+	}
+	return gov
```
Comment (mchirico) [inline pkg/analysis/analysis.go]: Returning gov; but, function has no return types defined.

File: pkg/analysis/analysis.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package analysis
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mchirico/go-gov/pkg/gov/house"
+	"github.com/mchirico/go-gov/pkg/httputils"
+)
+
+func GetHouseMembers(){
```
Comment (mchirico) [inline pkg/analysis/analysis.go]: No return types defined.

**Comment by ZoeChiri** [comment]
package senate

import (
	"encoding/json"
	"fmt"
	"github.com/mchirico/go-gov/pkg/httputils"
	"testing"
)

func TestSenate(t *testing.T) {
	url := "https://api.propublica.org/congress/v1/116/senate/members.json"

	key := "X-API-Key"
	value := "1vtlJSvzaaB6bTjJKzyakYnjnxrRzM22Ex3j2SDR"

	h := httputils.NewHTTP()
	h.Header(key, value)

	r, err := h.Get(url)

	var gov Senate
	err = json.Unmarshal(r, &gov)
	if err != nil {
		fmt.Println("error:", err)
	}

	m := map[string]int{}
	for _, v := range gov.Results[0].Members {
		m[v.State] += 1
	}

	for k, v := range m {
		if v < 1 {
			t.Fatalf("Data bad: %v,%v\n", k, v)
		}
	}

	fmt.Println(gov.Results[0].Members[0].FacebookAccount)

}
func TestGetSenate(t *testing.T) {
	gov, err := GetSenate()
	if err != nil {
		t.Fatalf("we got error")
	}

}

**Review by mchirico — APPROVED** [review]
Approved... please see changes.

**Pull request merged**